build-scripts/build_rule_playbooks.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python3
+
+import argparse
+import ssg.playbook_builder
+
+
+def parse_args():
+    p = argparse.ArgumentParser()
+    p.add_argument(
+        "--input-dir", required=True, dest="input_dir",
+        help="Input directory that contains all Ansible remediations"
+        "snippets for the product we are building. "
+        "e.g. ~/scap-security-guide/build/fedora/fixes/ansible"
+    )
```

Comment:
If we know the product YAML, we think that we also know where to look for the input directory - it is `build/<product-that-we-know>/fixes/ansible`. This means that we don't have to require this, because we know the default.

Reply:
OK, I'm not sure if we will need them when the script is used standalone, so I created an issue as you suggested. See https://github.com/ComplianceAsCode/content/issues/4034